docs/site/TEMP-TEXT-FILES/unit-names-and-patterns.txt

```diff
@@ -35,17 +51,21 @@ Inflected alternatives. If it does list inflected alternatives, you should look
 Length / Kilometer / long-other => {0} kilometers
 Area / Square-Kilometer / long-other => {0} square kilometers
 The pattern for power2 should be constructed so that if you take the word for "kilometers" and substituted it into the pattern, you get "square kilometers". So let's take an example from French:
-1. https://st.unicode.org/cldr-apps/v#/fr/Length/52e5f7e1046696e8 => {0} kilomètres
-2. https://st.unicode.org/cldr-apps/v#/fr/Area/18a3be1b1257d916 => {0} kilomètres carrés
+https://st.unicode.org/cldr-apps/v#/fr/Length/52e5f7e1046696e8 => {0} kilomètres
+https://st.unicode.org/cldr-apps/v#/fr/Area/18a3be1b1257d916 => {0} kilomètres carrés
 So the appropriate pattern for power2 would be:
-3. https://st.unicode.org/cldr-apps/v#/fr/CompoundUnits/15b049cba8052719 => {0} carrés
+https://st.unicode.org/cldr-apps/v#/fr/CompoundUnits/15b049cba8052719 => {0} carrés
```

Comment:
Converted these into actual number/bullet point lists instead of numbers and intents